radix-engine/src/model/resource_def.rs

```diff
@@ -36,17 +29,74 @@ pub enum ResourceDefError {
     },
 }
 
+#[derive(Debug, Clone, TypeId, Encode, Decode)]
+enum Flag {
+    IsSet(u64),
+    IsNotSet(u64),
+    True,
+}
```

Comment:
The `Flag` is misleading to me. I'd also suggest to name it `Condition` or even `FlagCondition`?

```rust
Condition {
    FlagSet(u64),
    FlagNotSet(u64),
    Any
}
```

Reply:
good point